File: winter-break-challenges/sumOfArray.java
public class sumOfArray {
    public static void main (String[] args) {
        int[] array = {1, 3, 6, 15, 78, 94};
        int sum = 0;

        for (int i: array) {
            sum += i;
        }

        System.out.println(sum);
    }
}

File: winter-break-challenges/sumOfArray.c
#include <stdio.h>

int main (void) {
    int array[] = {1, 3, 6, 15, 78, 94};
    int sum = 0;

    for (int i = 0; i < (sizeof(array) / sizeof(array[0])); i++) {
        sum += array[i];
    }

    printf("%d\n", sum);

    return 0;
}